sdk/src/wallet/operations/syncing/transactions.rs

```diff
@@ -118,96 +118,95 @@ where
+                                    Some(self.client().block_id(&included_block).await?),
+                                    // block metadata was Conflicting, but it's confirmed in another attachment
+                                    InclusionState::Confirmed,
+                                    &mut updated_transactions,
+                                    &mut spent_output_ids,
+                                );
+                            } else {
+                                log::debug!("[SYNC] conflicting transaction {transaction_id}");
+                                updated_transaction_and_outputs(
+                                    transaction,
+                                    None,
+                                    InclusionState::Conflicting,
+                                    &mut updated_transactions,
+                                    &mut spent_output_ids,
+                                );
+                            }
                         }
+                        // Do nothing, just need to wait a bit more
+                        TransactionState::Pending => {}
+                    }
+                    // else if input_got_spent {
+                    //     process_transaction_with_unknown_state(
+                    //         &*self.ledger().await,
+                    //         transaction,
+                    //         &mut updated_transactions,
+                    //         &mut output_ids_to_unlock,
+                    //     )?;
+                    // }
+                }
+                Err(ClientError::Node(crate::client::node_api::error::Error::NotFound(_))) => {
+                    if input_got_spent {
+                        process_transaction_with_unknown_state(
+                            &*self.ledger().await,
+                            transaction,
+                            &mut updated_transactions,
+                            &mut output_ids_to_unlock,
+                        )?;
                     }
-                    Err(e) => return Err(e.into()),
                 }
-            } else if input_got_spent {
-                process_transaction_with_unknown_state(
-                    &*self.ledger().await,
-                    transaction,
-                    &mut updated_transactions,
-                    &mut output_ids_to_unlock,
-                )?;
-            } else {
-                // Reissue if there was no block id yet, because then we also didn't burn any mana
-                log::debug!("[SYNC] reissue transaction {}", transaction.transaction_id);
-                let reissued_block = self
-                    .submit_signed_transaction(transaction.payload.clone(), None)
-                    .await?;
-                transaction.block_id.replace(reissued_block);
-                updated_transactions.push(transaction);
+                Err(e) => return Err(e.into()),
             }
+
+            //  else if input_got_spent {
```

Comment:
What's the plan?

Reply:
@Thoralf-M will tell me if he wants me to keep them or remove them, he has a PR touching this exact same code

Reply:
Please keep it commented and I will remove what's not needed in my PR

Reply:
👍🏻